sql/src/test/java/io/crate/integrationtests/ColumnPolicyIntegrationTest.java

```diff
@@ -148,6 +148,7 @@ public void testInsertNewColumnTableDynamic() throws Exception {
     }
 
     @Test
+    @UseRandomizedSession(schema = false)
     public void testInsertArrayIntoDynamicTable() throws Exception {
```

Comment:
Why is `schema = false` required here?

Reply:
the tests uses the ES API afterwards to verify that the mapping of the table is correct `getSourceMap()` --> this defaults to `doc` schema if nothing else specified. If we want to use our randomized schema it would be `getSourceMap(defaultSchema + ".dynamic_table")`

Reply:
I think it would be a nice addition to have an end-to-end test for the random schema. Maybe we can have a default method for checking the index name. If it turns out to be too tricky, we can also skip these tests for now.